### ISSUE TITLE
display errors after clicking "install abapGit repo"

### ISSUE THREAD

**Issue by larshp** [abapGit/abapGit#2795]
display errors after clicking "install abapGit repo"

I guess this also does not show errors anymore?

**Comment by larshp** [comment]
fixed in https://github.com/larshp/abapGit/pull/2798

**Comment by larshp** [comment]
oh, no, reopen

**Comment by danieltoro165051** [comment]
Hola, Ejecutè el boton instala abapGit y se muestra el siguiente error: HTTP error 110 occured: HTTPIO_PLG_CANCELED

Volví a ejecutar la opción instalar abapGit por segunda vez y ya no muestra nada pero tampoco se instala en $ abapgit

Note: The BASIS team is currently supporting me by configuring the certificates and testing the ZABAPGIT_TEST_SSL report because the ssl check does not currently work.

**Comment by larshp** [comment]
try manually creating the online project, https://docs.abapgit.org/guide-online-install.html

**Comment by striezl** [comment]
Hi, 

I just ran in to the issue as already described here: Clicked on install abapGit repo, but nothing happened. Searched for a while until I found these helpful hints, thanks a lot! 

In the end I solved the problem by deleting contents of table ZABAPGIT. My first attempt to install failed due to missing proxy. Nevertheless an entry in ZABAPGIT which seemed to prevent another installation has been made. 

I think we could save people some time and make the abapGit experience even greater if we either implement a proper error handling at that place or make sure that we only write to tab ZABAPGIT if the installation has suceeded or do both :-)

Thanks!

Kind regards

Bernhard
